packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -726,7 +727,7 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
 
       if (column != null) {
         actions.push(
-          onContextMenu({
+          ...onContextMenu({
```

Comment:
Was this a bug from incorrect types? Just making sure it's intentional since it was not spread before

Reply:
Yup